addons/binding/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/handler/OpenUVBridgeHandler.java

```diff
@@ -0,0 +1,164 @@
+                .create();
+    }
+
+    @Override
+    public void initialize() {
+        boolean validConfig = false;
+
+        logger.debug("Initializing OpenUV API bridge handler.");
+        Configuration config = getThing().getConfiguration();
+
+        OpenUVJsonResponse result = null;
+        String errorDetail = null;
+
+        // Check if an api key has been provided during the bridge creation
+        if (StringUtils.trimToNull((String) config.get(OpenUVBindingConstants.APIKEY)) == null) {
+            error += " Parameter 'apikey' must be configured.";
+            statusDescr = "Missing API key";
+        } else {
+            setApikey((String) config.get(OpenUVBindingConstants.APIKEY));
+            // Check if the provided api key is valid for use with the OpenUV service
+            try {
+                // Run the HTTP request and get the JSON response
+                String response = null;
+
+                try {
+                    Properties header = new Properties();
+                    header.put("x-access-token", getApikey());
+
+                    response = HttpUtil.executeUrl("GET", BASE_URL + "?lat=0&lng=0", header, null, null, 2000);
+                    logger.debug("apiResponse = {}", response);
+                    // Map the JSON response to an object
+                    result = gson.fromJson(response, OpenUVJsonResponse.class);
+                    if ((result.getError() != null) && (result.getError().equals("Invalid API Key"))) {
+                        error = "API key has to be fixed";
+                        errorDetail = result.getError();
+                        statusDescr = "Error : Invalid Key";
+                    } else {
+                        validConfig = true;
+                    }
+
+                } catch (IllegalArgumentException e) {
+                    errorDetail = e.getMessage();
+                    statusDescr = "@text/offline.uri-error";
+                }
+            } catch (IOException e) {
+                error = "Error running OpenUV API request";
+                errorDetail = e.getMessage();
+                statusDescr = "@text/offline.comm-error-running-request";
+            }
+        }
+
+        // Updates the thing status accordingly
+        if (validConfig) {
+            updateStatus(ThingStatus.ONLINE);
+        } else {
+            error = error.trim();
+            logger.debug("Disabling thing '{}': Error '{}': {}", getThing().getUID(), error, errorDetail);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, statusDescr);
+        }
+    }
+
+    public @Nullable String getApikey() {
+        return apikey;
+    }
+
+    public void setApikey(String apikey) {
+        this.apikey = apikey;
+    }
+
+    public Gson getGson() {
+        return this.gson;
+    }
+
+    @Override
+    public void handleCommand(@NonNull ChannelUID channelUID, @NonNull Command command) {
```

Comment:
You should annotate the class with `@NonNullByDefault` and use `@Nullable`. 
Using `@NonNull` shouldn't be necessary that way.